Updates to AnnData contents based on new scanpy compatibility

allyhawkins opened the pull request:
Closes #337 

Related to changes described in https://github.com/AlexsLemonade/scpca-nf/issues/773 and made in https://github.com/AlexsLemonade/scpca-nf/pull/774. 

Here I'm accounting for the new changes to contents in the individual and merged AnnData objects. Both objects have the following changes: 

- `X_PCA` and `X_UMAP` have been renamed to `X_pca` and `X_umap`
- `highly_variable` is now a column in the `.var` slot indicating if a gene is part of the HVG list 

Just the individual objects will have: 

- A new `.uns["pca"]` object that contains information about parameters and variance for PCA. I didn't describe every element of this, because I think if someone is using it they know what they're doing. Let me know if the explanation that I included is okay or if I should add more detail. 

I made the changes in the SCE contents file (in the AnnData section) and the merged object file. I think that should be everything.

sjspielman reviewed (approved):
All the changes here look fine to me, but I found one more spot to update. Don't need to see again though!

- This needs to be updated:
https://github.com/AlexsLemonade/scpca-docs/blob/f71da743780120d1a85a517150ac0c6b56ff7040/docs/getting_started.md?plain=1#L147-L153

- This _could_ be expanded to say something about `highly_variable` but probably isn't _necessary_. Up to you! https://github.com/AlexsLemonade/scpca-docs/blob/f71da743780120d1a85a517150ac0c6b56ff7040/docs/getting_started.md?plain=1#L175-L180

allyhawkins commented:
Thanks for catching that! I updated the PCA/UMAP names in the getting started doc, and I added a small sentence about the highly variable column. I'm still 50/50 on if we need it, but I don't think it can hurt. @sjspielman can you take a look at that change in https://github.com/AlexsLemonade/scpca-docs/pull/338/commits/f16c8b0e75f21d38b7cdbb0f421a499c1bcb2d9a.

sjspielman commented on docs/getting_started.md:
````diff
@@ -179,6 +179,8 @@ This list can be accessed using the following command in the `AnnData` objects:
 processed_adata.uns["highly_variable_genes"]
 ```
 
+Additionally, the `AnnData` objects contain a column in the `.var` slot, `"highly_variable"` indicating whether or not a gene is found in the list of highly variable genes.
````
Doesn't hurt!
